v2/test/plugins/test_cache.py

```diff
@@ -21,9 +21,23 @@
 
 from ansible.compat.tests import unittest
 from ansible.plugins.cache.base import BaseCacheModule
-from ansible.plugins.cache.memcached import CacheModule as MemcachedCache
 from ansible.plugins.cache.memory import CacheModule as MemoryCache
-from ansible.plugins.cache.redis import CacheModule as RedisCache
+
+SKIP = {}
+try:
+    import memcached
+except ImportError:
+    SKIP['memcached'] = True
+else:
+    # Use an else so that the only reason we skip this is for lack of
+    # memcached, not errors importing the plugin
+    from ansible.plugins.cache.memcached import CacheModule as MemcachedCache
```

Comment:
I would separate the try-except-else blocks to improve readability of the tests.